content/en/docs/topics/kubernetes_apis.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Kubernetes APIs"
+description: "Explains Kubernetes APIs in Helm templates"
+aliases: ["docs/k8s_apis/"]
+---
+
+Helm templates specify a [Kubernetes API group](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-groups) when defining a Kubernetes object, similar to a Kubernetes manifest file. It is specified in the `apiVersion` field of the template and it identifies the API version of the object.
+
+Like all API-driven system, Kubernetes has a [deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/) for its APIs. This means that after a certain number of releases, an API version cited for deprecation will no longer work in a Kubernetes version.
```

Comment:
`...an API version cited for deprecation will no longer work...`
maybe should be
`...an API version cited as deprecated will no longer work...`
or even simpler
`...a deprecated API version will no longer work`

Reply:
Updated